### PR TITLE
Updated tests to use CrateDb 2.1.7.

### PR DESCRIPTION
**Pull request by christianbader**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -5,6 +5,8 @@ Changes for crate
 Unreleased
 ==========
 
+ - Updated tests to use CrateDb 2.1.7.
```
Comment (chaudum) [inline CHANGES.txt]: CrateDB

Comment (chaudum) [reply CHANGES.txt]: tests do not require a changelog entry

File: src/crate/testing/layer.py [hunk 2]
```diff
@@ -44,7 +44,7 @@
 
 CRATE_CONFIG_ERROR = 'crate_config must point to a folder or to a file named "crate.yml"'
 HTTP_ADDRESS_RE = re.compile(
-    '.*\[http +\] \[.*\] .*'
+    '.*\[i.c.p.h.CrateNettyHttpServerTransport] \[.*\] .*'
```
Comment (mfussenegger) [inline src/crate/testing/layer.py]: Should we keep compat here for the older format? If not it's kind of a breaking changes for the testing layer.

Comment (christianbader) [reply src/crate/testing/layer.py]: Yes, good point

Comment (chaudum) [reply src/crate/testing/layer.py]: I agree, we should keep it compatible.

**Pull request merged**